Software/scan_key/AuthenticationSQL.py

```diff
@@ -24,19 +24,21 @@ def machineAuth(id, cursor):
     for auth in authdata:
         if(auth[machineType] != 1):
             print ("USER IS NOT AUTHORIZED\n")
+            os.system("omxplayer deny.wav &")
```

Comment:
Okay I see where you're going with that. Have any current ideas on how we will implement the wrapper?